context.go

```diff
@@ -93,7 +93,7 @@ func (nc *Conn) oldRequestWithContext(ctx context.Context, subj string, data []b
 	inbox := NewInbox()
 	ch := make(chan *Msg, RequestChanLen)
 
-	s, err := nc.subscribe(inbox, _EMPTY_, nil, ch)
+	s, err := nc.subscribe(inbox, _EMPTY_, nil, ch, true)
```

Comment:
Are you sure? We did not set it to SyncSubscription for this one, so should it be `false` here?

Reply:
If we create the chan I was setting it to sync, it would have been set to a chanSub type, but I think that was wrong. Has no effect.

Reply:
I see..